pkg/registry/shadow/template/rest.go

```diff
@@ -100,6 +116,82 @@ type REST struct {
 
 	// is this Rest for CRD resource.
 	CRD *apiextensionsv1.CustomResourceDefinition
+
+	mcLister            clusterlisters.ManagedClusterLister
+	childClsClientCache ChildClsClient
+}
+
+// resourceInCCls the resource need get from child cluster
+var resourceInCCls = []string{Pods, events}
+
+// resourceInCCls need to get resource from child cls
+func (r *REST) resourceInCCls() bool {
+	resource, _ := r.getResourceName()
```

Comment:
Currently only pods and events are allowed, but what if users want to list CRs in child clusters in the future?